code/_core/mob/movement.dm

```diff
@@ -56,7 +56,7 @@
 		. *= 2 - (health.health_current/health.health_max)
 
 
-/mob/Move(NewLoc,Dir=0,step_x=0,step_y=0)
+/mob/Move(NewLoc,Dir=0,step_x=0,step_y=0,var/control_move = TRUE)
```

Comment:
I don't like adding control_move to this as there are other solutions. Maybe go into the ``chosenObserver`` loop and add 
```
if(chosenObserver.move_dir)
 observed.observers -= src
 observed = null
 continue
```